product_docs/docs/edbcloud/beta/using_cluster/connecting_your_cluster/01_connecting_from_azure/02_virtual_network_peering.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Virtual Network Peering Example
+---
+
+These are the steps to connect using Virtual Network Peering. To walk you through this example in your own environment, see the prerequisites described in [Connection Walkthrough Examples](../../connecting_your_cluster/#connection-walkthrough-examples).
+
+#### Step 1: Create virtual network peering
+
+You need to add two peerings links from both the VM VNet `vnet-client` side
+and your EDB Cloud VNet `vnet-japaneast` side. One approach is to add the links from the VM's VNet `vnet-client` side,
```

Comment:
We should just tell them to create them from the `vnet-client` side, rather than even suggesting that there is another approach. /cc @zozowell